r3bbase/pars/R3BMSOffsetFinder.h

```diff
@@ -0,0 +1,92 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// -------------------------------------------------------------------------
+// -----         R3BMSOffsetFinder source file                         -----
+// -----             Created 29/05/18  by J.L. Rodriguez-Sanchez       -----
+// -----             Adapted 26/07/23 by L.M. Fonseca                  -----
+// -------------------------------------------------------------------------
+
+#pragma once
+
+#include "FairTask.h"
+#include "TH1F.h"
+
+class TClonesArray;
+class R3BMSOffsetPar;
+class R3BEventHeader;
+class R3BSamplerMappedData;
+class R3BMSOffsetFinder : public FairTask
+{
+
+  public:
+    /** Default constructor **/
+    R3BMSOffsetFinder();
+
+    R3BMSOffsetFinder(const R3BMSOffsetFinder&) = delete;
+    R3BMSOffsetFinder(R3BMSOffsetFinder&&) = delete;
+    R3BMSOffsetFinder& operator=(const R3BMSOffsetFinder&) = delete;
+    R3BMSOffsetFinder& operator=(R3BMSOffsetFinder&&) = delete;
+    /** Standard constructor **/
+    explicit R3BMSOffsetFinder(const TString& name, Int_t iVerbose = 1);
+
+    /** Destructor **/
+    ~R3BMSOffsetFinder() override;
+
+    /** Virtual method Init **/
+    InitStatus Init() override;
+
+    /** Virtual method Exec **/
+    void Exec(Option_t* opt) override;
+
+    /** Virtual method FinishEvent **/
+    void FinishEvent() override;
+
+    /** Virtual method FinishTask **/
+    void FinishTask() override;
+
+    /** Virtual method Reset **/
+    virtual void Reset();
+
+    /** Virtual method ReInit **/
+    InitStatus ReInit() override;
+
+    /** Virtual method Search MS Offset **/
+    virtual void SearchMSOffset();
+
+    /** Virtual method SetParContainers **/
+    void SetParContainers() override;
+
+    /** Accessor functions **/
+    [[nodiscard]] Double_t GetMSOffset() const { return fMSOffset; }
+
+    void SetMSOffset(Double_t MSOffset) { fMSOffset = MSOffset; }
+
+    void SetMinStadistics(Int_t MinStadistics) { fMinStadistics = MinStadistics; }
```

Comment:
SetMinStatistics, s/d/t/
https://en.wiktionary.org/wiki/statistics

Reply:
You are right, being changed right now.